lib/token.js

```diff
@@ -391,7 +391,7 @@ function buildAuthorizeUrl(sdk, oauthParams) {
  *
  *    Optional:
  *      redirectUri: defaults to window.location.href
- *      scopes: defaults to ['openid', 'email']
+ *      scope: defaults to ['openid', 'email']
```

Comment:
Cool. I can't change the query param, because that's part of the spec. However, the **scope** query param is a space-separated list, so it makes sense that we'd have a different name for an array version (**scopes**).